src/transformers/utils/bitsandbytes.py

```diff
@@ -272,21 +272,25 @@ def get_keys_to_not_convert(model):
     if (not has_tied_params) and is_base_model:
```

Comment:
I found out that checking `base_model_prefix` for base model might be unreliable. I wonder what is the risk for treating a base model as a chat/instruct model in quantization?

Reply:
If I understand correctly your question is about what is the risk of quantizing a standalone base model? If that's the case there shouldn't be any major risk I think

Reply:
I think so too.  Then may be there is no need to do checking there.